src/fides/core/dataset.py

```diff
@@ -373,3 +377,33 @@ def generate_dynamo_db_datasets(aws_config: Optional[AWSConfig]) -> Dataset:
     described_dynamo_tables = describe_dynamo_tables(client, dynamo_tables)
     dynamo_dataset = create_dynamodb_dataset(described_dynamo_tables)
     return dynamo_dataset
+
+
+def get_snowflake_schemas(
+    engine: Engine,
+) -> Dict[str, Dict[str, List[str]]]:
+    """
+    Returns Datasets that match the case-sensitivity that may be
+    required by Snowflake.
+    """
+    schema_cursor = engine.execute(
+        text("SHOW /* sqlalchemy:get_schema_names */ SCHEMAS")
```

Comment:
sort of piling on the comment above -- the comment block (i assume that the `/*` indicates a comment!) just confuses me a bit. is there a reason it comes _within_ the `SHOW SCHEMAS` command?

again, this could very well just be my lack of experience speaking here 😳

Reply:
No I thought about removing the comments but ended up liking the paper trail. This is primarily my wanting to keep it as common as possible with the existing `sqlalchemy` connector. Ideally, I'd like to open a PR against `snowflake-sqlalchemy` to resolve it and I liked the idea of the breadcrumbs there but it does feel a bit out of place looking at it again

Reply:
oh i think the paper trail makes sense, i was _thinking_ it was something like that. maybe just an explanatory (python-level 😄 ) code comment to clarify that? or does that feel too verbose? you could throw it in wherever you end up adding the additional context discussed above maybe...

anyway, i get the motivation with the paper trail, so not suggesting you remove it! just wanted to double check on this and see if we can make it a little bit more explicit for someone re-looking at this down the road.

Reply:
I just ended up removing the code comments inline and adding a lot more into the docs block. It felt like there is a chance those extra comments could get stale and not be as meaningful anyways so hopefully that should be enough. Also, best next move is likely sorting out the actual connector with a PR so we can step away from this change here 🤞🏽